app.js

```diff
@@ -11,6 +11,8 @@ app.get('/api/topics', getTopics)
 
 app.get('/api', getEndpoints)
 
+app.get('/api/articles/', getArticle)
```

Comment:
As we are wanting multiple articles, the controller being called getArticle (singular) does not make sense. Therefore, we need separate controllers and models for the two endpoints ( /api/articles/ and '/api/articles/:article_id')